src/BLSApkRegistry.sol

```diff
@@ -118,33 +116,33 @@ contract BLSApkRegistry is BLSApkRegistryStorage {
         );
 
         // H(m) 
-        BN254.G1Point memory messageHash = getMessageHash(msg.sender);
+        BN254.G1Point memory messageHash = getMessageHash(operator);
```

Comment:
I wonder if the message hash should use the registry coordinator rather than the BLSApkRegistry, since the registry coordinator is "the AVS" (as far as the Dmgr is concerned, anyway)

Reply:
i guess we're registering within the context of the AVS though so maybe it makes sense as-is

Reply:
> i guess we're registering within the context of the AVS though so maybe it makes sense as-is

yeah I agree with this.
will make the message signature more EIP712-compatible, but I think using the BLSApkRegistry address here is OK